src/config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   basePath: process.env.BASE_PATH || process.cwd(),
   dataPath: process.env.DATA_PATH || 'archive',
-  fileName: process.env.FILE_NAME_FORMAT || 'archive_<%=date%>',
+  fileName: process.env.FILE_NAME_FORMAT || 'archive_{{date}}',
```

Comment:
should we go ahead and stick a `.tgz` on here so we're uploading things with an extension by default?